src/game/camera.c

```diff
@@ -1606,15 +1605,19 @@ void mode_parallel_tracking_camera(struct Camera *c) {
  * Fixed camera mode, the camera rotates around a point and looks and zooms toward Mario.
  */
 void mode_fixed_camera(struct Camera *c) {
+#ifdef DISABLE_LEVEL_SPECIFIC_CHECKS
+    set_fov_function(CAM_FOV_APP_45);
+#else
```

Comment:
don't remove the ability to set FOV with fixed cameras

```suggestion
#ifndef DISABLE_LEVEL_SPECIFIC_CHECKS
```

Reply:
blocker